data/test-cases/plugins/MS Teams sync/Message sync/Non-connected_user.md

```diff
@@ -0,0 +1,47 @@
+---
+# (Required) Ensure all values are filled up
+name: "Message from non-connected user"
```

Comment:
should we also add a test case when a non-connected user messages in DM/GM or will that be present in a different test file?

Reply:
That will be covered under plugin configs